src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/ProjectModifier.cs

```diff
@@ -69,50 +63,65 @@ public async Task AddAuthCodeAsync()
                 IsMinimalApp = isMinimalApp
             };
 
-            var filteredFiles = codeModifierConfig.Files.Where(f => ProjectModifierHelper.FilterOptions(f.Options, options));
             //Go through all the files, make changes using DocumentBuilder.
-            foreach (var file in filteredFiles)
+            foreach (var file in codeModifierConfig.Files.Where(f => ProjectModifierHelper.FilterOptions(f.Options, options)))
```

Comment:
I'm really not a fan of making the foreach loop longer as such. Not sure what the dotnet convention is but lets look into it. I'll add another comment here when I find something.

Reply:
Yeah I'm cool with that :)

Reply:
I couldn't easily find anything but I feel for readability and debugging sake, lets keep it like it was.
Thanks :)

Reply:
Agreed, I actually noticed yesterday that I had made debugging harder for myself by doing it that way 😆